internal/pkg/rpaas/manager.go

```diff
@@ -252,6 +262,43 @@ type CertificateData struct {
 	Key         string `json:"key"`
 }
 
+func getAnnotations(params map[string]interface{}) (annotations map[string]string) {
+	p, found := params["annotations"]
+	if !found {
+		return
+	}
+	err := json.Unmarshal([]byte(p.(string)), &annotations)
+	if err != nil {
+		return
+	}
```

Comment:
Just a niptick. `p.(string)` might panic against a mal-formed/malicious payload. 

```suggestion
	annotationsStr, ok := p.(string)
	if !ok {
		return
	}

	err := json.Unmarshal([]byte(annotationsStr), &annotations)
	if err != nil {
		return
	}
```

Reply:
Ouch! Thanks, I didn't notice it